src/hooks/useChapterId.ts

```diff
@@ -12,7 +12,7 @@ import { getChapterIdsForJuz, getChapterIdsForPage } from 'src/utils/chapter';
  * - /page/2 => ["2"]
  * - /chapter/1 => ["1"]
  */
-const useChaptersIdByUrlPath = (): string[] => {
+const useChaptersIdsByUrlPath = (): string[] => {
```

Comment:
I think this should be `useChapterIdsByUrlPath`

Reply:
✅